src/WebGPU/programs/pickTexture/getProgram.ts

```
@@ -0,0 +1,106 @@
+import shaderCode from "./shader.wgsl"
+ 
+const STRIDE = 4 + 2 + 1
+
+export default function getProgram(
+  device: GPUDevice,
+  debugPresentationFormat: GPUTextureFormat
```

ESLint

[error] 'debugPresentationFormat' is defined but never used.

(@typescript-eslint/no-unused-vars)